.github/ISSUE_TEMPLATE/documentation-update.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: Documentation request
+about: Use this to propose information improvements and additions
```

Comment:
Suggest an improvement/addition in the Keploy Server Docs.

Reply:
Done